tests/unit/commands/repos_test.py

```diff
@@ -77,11 +77,11 @@ def test_override_repos_from_directory(self):
     def test_update_managed_repos(self, fake_update_local_repo):
         activate_bundle(['bundle-a'])
         update_managed_repos()
-        fake_update_local_repo.assert_called_once_with()
+        fake_update_local_repo.assert_has_calls([call(), call()])
 
     @patch('dusty.source.Repo.update_local_repo')
     def test_update_managed_repos_for_both(self, fake_update_local_repo):
         activate_bundle(['bundle-a'])
         activate_bundle(['bundle-b'])
         update_managed_repos()
-        fake_update_local_repo.assert_has_calls([call(), call()])
+        fake_update_local_repo.assert_has_calls([call(), call(), call()])
```

Comment:
really weird that these did not start failing till i actually changed the values in here (these tests seem to never have been correct)